nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -72,15 +73,120 @@ in {
           https://github.com/PipeWire/pipewire/blob/master/NEWS
         '';
       };
+
+      extraConfig = {
+        pipewire = mkOption {
+          type = lib.types.attrsOf json.type;
+          default = {};
+          example = {
+            "10-clock-rate" = {
+              "context.properties" = {
+                "default.clock.rate" = 44100;
+              };
+            };
+            "11-channelmix-upmix" = {
+              "stream.properties" = {
+                "channelmix.upmix" = false;
+              };
+            };
+          };
+          description = lib.mdDoc ''
+            The pipwwire config can be used to configure the PipeWire server, including adding Virtual Devices and filter chains.
```

Comment:
"pipwwire"?

Reply:
Also, weird capitalization all over. Also, I was thinking about something more like "every attrset becomes a separate drop-in file in /etc/pipewire/pipewire.conf.d".

Reply:
The PipeWire website and git repo use  `PipeWire` so i changed all of them to that

Reply:
Added the bit about separated files